src/OrbitGl/ManualInstrumentationManager.h

```diff
@@ -37,8 +37,7 @@ class ManualInstrumentationManager {
 
  private:
   absl::flat_hash_set<AsyncTimerInfoListener*> async_timer_info_listeners_;
-  absl::flat_hash_map<uint32_t, orbit_client_protos::TimerInfo> async_timer_info_start_by_id_;
```

Comment:
Why are you removing this line?

Reply:
It's unused.

Reply:
Right, sorry I confused that field with the similar one in ApiEventProcessor.

Reply:
nobody uses this field